editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -2786,7 +2797,8 @@ void Node3DEditorViewport::_draw() {
 							get_theme_icon("ViewportZoom", "EditorIcons"),
 							get_theme_font("font", "Label"),
 							get_theme_font_size("font_size", "Label"),
-							vformat("%s u", String::num(cursor.distance).pad_decimals(precision)));
+							vformat("%s u", String::num(cursor.distance).pad_decimals(precision)),
+							zoom_failed_attempts_count > 15);
```

Comment:
This number is arbitrary, and the optimal number will likely vary between systems. 15 seems good to me, but maybe it should be higher or lower. A lower number will make the message show up sooner, which might be good or annoying.